lib/custom/modes/apache.js

```diff
@@ -8,8 +8,8 @@ const CodeMirror = require('codemirror')
 CodeMirror.defineMode('apache', function(config) {
   return {
     token: function(stream, state) {
-      var sol = stream.sol() || state.afterSection
-      var eol = stream.eol()
+      const sol = stream.sol() || state.afterSection
```

Comment:
Please remove `/* eslint-disable */` from this file and fix errors before merging